layouts/partials/footer_section.html

```diff
@@ -3,8 +3,9 @@
     display: block;
     margin-left: auto;
     margin-right: auto;
-    width: 50%;
-    padding: 30px;
+    width: 495px;
+    max-width: 100%;
+    padding: 30px 0;
```

Comment:
Just have a preview in on mobile and thsi works fine :)
And what's the meaning of `495px` and `30px 0`？

Reply:
Returned padding but enlarged the picture. This is done so that they do not differ from the main site.